main.js

```diff
@@ -62,3 +62,72 @@ cancelbtn.addEventListener('click', cancel);
 ulLinks.addEventListener('click', () => { unfixHeader(); cancel(); });
 ulLinks2.addEventListener('click', () => { unfixHeader(); cancel(); });
 ulLinks3.addEventListener('click', () => { unfixHeader(); cancel(); });
+
+// popup detail project start
+const button = document.querySelectorAll('.btn2');
+const main = document.querySelector('.main');
+const footer = document.getElementById('footer');
+// cancelIcon Object
+const cancelIconInfos = {
+  src: 'template5Images/Icon-Cancel.svg',
+  alt: 'cancelIcon',
+};
+const Modalsection = document.createElement('section');
+Modalsection.classList.add('dis-none');
+Modalsection.innerHTML = `
```

Comment:
**Changes Required**
- [ ] **In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects**. The html is currently hard-coded and you need to use JavaScript to create elements and dynamically append them to the DOM. Let us please try to fix this as specified in the project requirements.